tests/TestCase/Network/Http/Adapter/StreamTest.php

```diff
@@ -158,6 +158,9 @@ public function testSendContextSsl()
             'ssl_verify_peer' => true,
             'ssl_verify_depth' => 9000,
             'ssl_allow_self_signed' => false,
+            'proxy' => [
+                'proxy' => '127.0.0.1:8080'
```

Comment:
No, because you need username and password key for credentials Header. It is all for the proxy, but the first half is for the Headers the second half is for the Stream context. 


You could rename the second key to e.g. `url`, but I thought it would be better, when the key matches with the key that is used by php.

On Mon, Aug 31, 2015 at 6:47 PM, Mark Story <notifications@github.com>
wrote:

>> @@ -158,6 +158,9 @@ public function testSendContextSsl()
>>              'ssl_verify_peer' => true,
>>              'ssl_verify_depth' => 9000,
>>              'ssl_allow_self_signed' => false,
>> +            'proxy' => [
>> +                'proxy' => '127.0.0.1:8080'
> Why two keys? Could we have just one instead?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cakephp/cakephp/pull/7326/files#r38331609